curvefs/proto/metaserver.proto

```diff
@@ -300,6 +303,21 @@ message MetaServerMetadata {
     // required uint32 checksum = 4;
 }
 
+message AppendS3ChunkInfoRequest {
+    required uint32 poolId = 1;
+    required uint32 copysetId = 2;
+    required uint32 partitionId = 3;
+    required uint32 fsId = 4;
+    required uint64 inodeId = 5;
+    map<uint64, S3ChunkInfoList> s3ChunkInfoAdd = 6;
+    map<uint64, S3ChunkInfoList> s3ChunkInfoRemove = 7;
```

Comment:
Need an explanation when it will be used

Reply:
remove is used for compaction